template.yml

```diff
@@ -94,13 +176,27 @@ Resources:
       Type: String
       Value: !Ref AWS::Region
 
-  DynamoDBTableParameter:
+  DynamoDBIssueTableParameter:
     Type: AWS::SSM::Parameter
     Properties:
-      Name: !Sub "/pagerduty-to-jira/${Stage}/DYNAMODB_TABLE"
+      Name: !Sub "/pagerduty-to-jira/${Stage}/DYNAMODB_ISSUE_TABLE"
       Type: String
       Value: !Ref IssueMappingTable
 
+  DynamoDBUserTableParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: !Sub "/pagerduty-to-jira/${Stage}/DYNAMODB_USER_TABLE"
+      Type: String
+      Value: !Ref UserMappingTable
+
+  DynamoDBOrganizationTableParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Name: !Sub "/pagerduty-to-jira/${Stage}/DYNAMODB_ORGANIZATION_TABLE"
+      Type: String
+      Value: !Ref OrganizationMappingTable
+
 Outputs:
   PagerDutyToJiraFunction:
     Description: "Lambda ARN"
```

Comment:
How was SAM? I think I like the serverless syntax better. Fewer capital letters.